releases/7.6.0.md

```diff
@@ -272,6 +272,58 @@ Minor incompatibilities:
         ```
         LOCALE_PATHS.insert(0, os.path.join(APP_ROOT, 'locale'))
         ```
+    4. Remove the `ARCHES_APPLICATIONS` setting. If your application is an Arches Application (designed for redistribution), you should create an AppConfig in apps.py with the class attribute `is_arches_application = True`.
+    5. Replace:
```

Comment:
We might be able to add some backward compatibility here by keeping the arches_applications argument as a synonym for the new installed_apps argument. I like the idea of doing that to remove a little friction.

Reply:
IMO I'd rather bite the bullet and deal with the friction rather than have the stray setting && small amount of tech debt it'll take to remove it later. Not a super strong opinion though

Reply:
They will have to change settings.py either way, the question is just whether to allow changing it in two fewer places, but after sitting with it for a bit it does have the potential to be confusing, so after sitting with it for a bit, it's probably better to not leave in the stray argument.